aurora/aurora-war/src/main/webapp/look/jsp/DomainsBar.jsp

```diff
@@ -56,7 +56,7 @@ LookHelper helper = LookHelper.getLookHelper(session);
 String spaceId    	= Encode.forUriComponent(request.getParameter("privateDomain"));
 String subSpaceId   = Encode.forUriComponent(request.getParameter("privateSubDomain"));
 String componentId  = Encode.forUriComponent(request.getParameter("component_id"));
-boolean displayPersonalSpace = StringUtil.getBooleanValue(request.getParameter("FromMySpace"));
```

Comment:
It should be natural here to also verify that current session is not an anonymous one, no?

Reply:
OK, anonymous user has guest access.